twitterFederation/services/comments/src/types/comment.ts

```diff
@@ -0,0 +1,25 @@
+import { gql } from 'apollo-server';
+
+export default gql`
+  type Comment {
+    id: ID!
+    user_id: ID!
+    tweet_id: ID!
+    comment: String!
+    createdAt: Date
+    updatedAt: Date
+    tweet: Tweet
+    user: User
+  }
+
+  input CreateCommentInput {
```

Comment:
Move the inputs definition to where you are using them. The mutation file in this case.